src/pages/signin/TermsAndLicenses.js

```diff
@@ -10,16 +10,8 @@ import LocalePicker from '../../components/LocalePicker';
 
 const TermsAndLicenses = props => (
     <>
-        <View
-            style={[
-                styles.dFlex,
-                styles.flexRow,
-                styles.flexWrap,
-                styles.textAlignCenter,
-                styles.alignItemsCenter,
-            ]}
-        >
-            <Text style={[styles.textAlignCenter, styles.textExtraSmallSupporting]}>
+        <Text style={[styles.textExtraSmallSupporting]}>
+            <Text style={[styles.textExtraSmallSupporting]}>
```

Comment:
Because our `<Text>` component has a `fontSize` prop with default value (15) we need to pass styles to each nested `<Text>` to override the default
If  `<Text>`  didn't have defaults - nested elements would have had same styles as parent

Though this seems the first time we have text in text used